### DISCUSSION TITLE
Welcome to AdClear Content Blocker for Android Discussions!

### DISCUSSION THREAD

**Discussion by DustinMorgan** [SEVENNetworks/AdClear-Content-Blocker-Open-Source-Issue-Tracker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WOAftp** [comment]
AMA

**Reply by sromee** [reply]
Good

**Reply by golfsxpro** [reply]
> Good

**Comment by golfsxpro** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
>

**Reply by golfsxpro** [reply]
@golfsxpro
